lib/devices/cpu_device_manager.js

```diff
@@ -51,12 +51,15 @@ export default class CpuDeviceManager {
 
 class CpuDevice {
   constructor(id, active=false) {
-    this.id = id
+    this.id = id;
```

Comment:
In 1.7.1 they are unmarshalling to go types and before having a number didn't break but now it throws an error that a number cannot be unmarshalled to a string